perma_web/static/bundles/create.js

```diff
@@ -1982,15 +1984,47 @@ webpackJsonp([1],[
 	  setCurrent: function setCurrent(orgId, folderIds) {
 	    var selectedFolders = ls.getAll();
 	    selectedFolders[current_user.id] = { 'folderIds': folderIds, 'orgId': orgId };
+	
+	    if (folderIds && folderIds.length) {
+	      history.pushState(null, null, "?folder=" + folderIds.join('-'));
+	    }
+	
 	    Helpers.jsonLocalStorage.setItem(localStorageKey, selectedFolders);
 	    Helpers.triggerOnWindow("CreateLinkModule.updateLinker");
 	  }
 	};
 	
+	function folderListFromUrl() {
+	  var queryDict = Helpers.getQueryStringDict();
+	  if (queryDict['folder']) {
+	    try {
+	      var folder_list = queryDict['folder'].split('-');
+	      folder_list = folder_list.map(function (s) {
+	        return parseInt(s);
+	      });
+	      folder_list.forEach(function (i) {
+	        if (isNaN(i)) throw "Invalid folder list";
+	      });
+	      return folder_list;
+	    } catch (err) {
+	      console.error(err);
```

Comment:
same comment here. Thinking we can punt on this until I fix up the ErrorHandler to be a bit more useful, then I can add to it a sweep for more js errors.